### PR TITLE
integrity hitl testing

### PR DESCRIPTION
**Pull request by kovach**
hit testing for https://github.com/swift-nav/libswiftnav/pull/156

### CODE REVIEW

**Comment by kovach** [comment]
retest this please

**Comment by kovach** [comment]
retest this please

**Comment by kovach** [comment]
retest this please

File: src/base_obs.c [hunk 1]
```diff
@@ -156,8 +159,9 @@ static void update_obss(obss_t *new_obss)
       }
       base_obss.has_pos = 1;
     } else {
+      // TODO(dsk) check for repair failure
```
Comment (fnoble) [inline src/base_obs.c]: `//` -> `/* */`

File: src/base_obs.c [hunk 2]
```diff
@@ -156,8 +159,9 @@ static void update_obss(obss_t *new_obss)
...
       /* There was an error calculating the position solution. */
-      log_error("Error calculating base station position (%ld)\n", ret);
+      log_warn("Error calculating base station position: (%ld). See solution.c or calc_PVT for info.\n", ret);
```
Comment (fnoble) [inline src/base_obs.c]: I'd axe this, its the kind of message that confuses our customers a lot.. If you want to make it better you could use the string error.

File: src/solution.c [hunk 3]
```diff
@@ -188,7 +191,7 @@ static void output_baseline(u8 num_sdiffs, const sdiff_t *sdiffs,
   case FILTER_FIXED:
     chMtxLock(&amb_state_lock);
     ret = dgnss_baseline(num_sdiffs, sdiffs, position_solution.pos_ecef,
-                         &amb_state, &num_used, b);
+                         &amb_state, &num_used, b, disable_raim, DEFAULT_RAIM_THRESHOLD);
```
Comment (fnoble) [inline src/solution.c]: Line length

File: src/solution.c [hunk 4]
```diff
@@ -203,7 +206,7 @@ static void output_baseline(u8 num_sdiffs, const sdiff_t *sdiffs,
     flags = 0;
     chMtxLock(&amb_state_lock);
     ret = baseline(num_sdiffs, sdiffs, position_solution.pos_ecef,
-                   &amb_state.float_ambs, &num_used, b);
+                   &amb_state.float_ambs, &num_used, b, disable_raim, DEFAULT_RAIM_THRESHOLD);
```
Comment (fnoble) [inline src/solution.c]: Line length

File: src/solution.c [hunk 5]
```diff
@@ -570,7 +568,7 @@ void process_matched_obs(u8 n_sds, gps_time_t *t, sdiff_t *sds)
       reset_iar = false;
     }
     /* Update filters. */
-    dgnss_update(n_sds, sds, position_solution.pos_ecef);
+    dgnss_update(n_sds, sds, position_solution.pos_ecef, disable_raim, DEFAULT_RAIM_THRESHOLD);
```
Comment (fnoble) [inline src/solution.c]: Line length

**Comment by fnoble** [comment]
Minor tweaks, otherwise looks ok

**Pull request merged**